lib/config/rules/jsx-a11y.js

```diff
@@ -19,28 +25,42 @@ module.exports = {
   'jsx-a11y/href-no-hash': 'error',
   // Enforce <html> element has lang prop.
   'jsx-a11y/html-has-lang': 'error',
-  // Enforce that <img> JSX elements use the alt prop.
-  'jsx-a11y/img-has-alt': 'error',
+  // Enforce iframe elements have a title attribute.
+  'jsx-a11y/iframe-has-title': 'error',
   // Enforce <img> alt prop does not contain the word "image", "picture", or "photo".
   'jsx-a11y/img-redundant-alt': 'warn',
+  // Enforce that elements with interactive handlers like onClick must be focusable.
+  'jsx-a11y/interactive-supports-focus': 'error',
   // Enforce that <label> elements have the htmlFor prop.
   'jsx-a11y/label-has-for': 'error',
+  // Enforces that <audio> and <video> elements must have a <track> for captions. <FEEDBACK>
+  'jsx-a11y/media-has-caption': 'off',
   // Enforce lang attribute has a valid value.
   'jsx-a11y/lang': 'error',
   // Enforce that onMouseOver/onMouseOut are accompanied by onFocus/onBlur for keyboard-only users.
   'jsx-a11y/mouse-events-have-key-events': 'error',
   // Enforce that the accessKey prop is not used on any element to avoid complications with keyboard commands used by a screenreader.
   'jsx-a11y/no-access-key': 'error',
+  // Enforce autoFocus prop is not used. <FEEDBACK>
+  'jsx-a11y/no-autofocus': 'off',
```

Comment:
I say turn it on because:
* Cross-browser `autofocus` has always been hella flaky for me, so it's good to get people thinking of alternatives
* This rule reflects [long standing sentiment of the React authors](https://github.com/facebook/react/issues/3066) that `autofocus` isn't reliable, and that they don't have  client/server use cases fleshed out yet